src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -174,9 +175,14 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
       //    + file:/C:/hyperspace/src/test/part-00003.snappy.parquet
       import spark.implicits._
       val dataPathColumn = "_data_path"
-      val lineageDF = fileIdTracker.getFileToIdMap.toSeq
+      val isDeltaLakeSource = DeltaLakeRuleUtils.isDeltaLakeSource(df.queryExecution.optimizedPlan)
```

Comment:
Can we move this to source provider somehow? Basically, we don't want source specific implementation outside the provider.

Reply:
question, why does delta not require the `replace`?

Reply:
input_file_neme() function of Delta returns "file:/" not "file:///"

Reply:
I see. How about we normalize the output of `input_file_name` by removing "file:/" or "file:///"? (and same to the `getFileToIdMap` keys)

Reply:
replacing filename - id map is cheaper than normalizing all file paths of all rows. I'll add an API for this.